week3/homework/exercises/2-authentication/script.js

```diff
@@ -8,8 +8,19 @@
  * Hints:
  * - for basic authentication the username and password need to be base64 encoded
  */
-function printBooks() {
+const fetch = require('node-fetch');
+
+async function printBooks() {
   // YOUR CODE GOES IN HERE
+  try {
+    const res = await fetch('https://restapiabasicauthe-sandbox.mxapps.io/api/books',
+    {headers: {'Authorization': 'Basic YWRtaW46aHZnWDhLbFZFYQ=='}}
+    );
```

Comment:
```suggestion
    const res = await fetch(
        'https://restapiabasicauthe-sandbox.mxapps.io/api/books',
        {headers: {'Authorization': 'Basic YWRtaW46aHZnWDhLbFZFYQ=='}}
    );
```
Take a look at your indentation